### PR TITLE
[CARBONDATA-403] Add example for data load without using kettle

### PR DESCRIPTION
**Pull request by jackylk**
In this PR, example SQL and dataframe usage is added for loading data without kettle

### CODE REVIEW

File: examples/src/main/scala/org/apache/carbondata/examples/CarbonExample.scala [hunk 1]
```diff
@@ -32,24 +32,34 @@ object CarbonExample {
...
            name String, phonetype String, serialname String, salary Int)
            STORED BY 'carbondata'
            """)
 
+    // load data using kettle
     cc.sql(s"""
            LOAD DATA LOCAL INPATH '$testData' into table t3
            """)
 
+    // load data without kettle
```
Comment (chenliang613) [inline examples/src/main/scala/org/apache/carbondata/examples/CarbonExample.scala]: suggest either create a different example for without kettle or do the different query respectively for two type of load data.

Comment (lion-x) [inline examples/src/main/scala/org/apache/carbondata/examples/CarbonExample.scala]: I quite agree with @chenliang613 . That is a better way.

**Comment by jackylk** [comment]
CI passed:
http://136.243.101.176:8080/job/ApacheCarbonManualPRBuilder/610/

**Comment by chenliang613** [comment]
LGTM

**Pull request closed without merging**